site2/website-next/docusaurus.config.js

```diff
@@ -349,10 +349,17 @@ module.exports = {
       <p>Copyright © ${new Date().getFullYear()} The Apache Software Foundation. All Rights Reserved. Apache, Pulsar, Apache Pulsar, and the Apache feather logo are trademarks or registered trademarks of The Apache Software Foundation.</p>`,
     },
     prism: {
-      // theme: lightCodeTheme,
-      // darkTheme: darkCodeTheme,
       theme: require("prism-react-renderer/themes/dracula"),
-      additionalLanguages: ["powershell", "java", "go", "c", "cpp", "python"],
+      additionalLanguages: [
+        "csharp",
+        "groovy",
+        "java",
+        "ini",
+        "powershell",
+        "properties",
+        "protobuf",
+        "yaml",
+      ],
```

Comment:
@dave2wave this change should be in this repo.

I pull in other changes so that reviewers can easily preview the final result. We carry changes under `docs` to the main repo to prevent regression by synching.

Reply:
For the upstream changes, see https://github.com/apache/pulsar/pull/17342.

Reply:
Related docusaurus guide:

* https://docusaurus.io/docs/markdown-features/code-blocks#supported-languages
* https://github.com/FormidableLabs/prism-react-renderer/blob/c1430679ee6cb0fb652d3a1b63f62f0c270ad631/src/vendor/prism/includeLangs.js
* https://prismjs.com/#supported-languages